inventory/any.cf

```diff
@@ -136,15 +136,21 @@ bundle agent cfe_autorun_inventory_memory
 # @brief Inventory the memory (Enterprise only)
 {
   vars:
-    enterprise::
+    enterprise.!aix::
       # due to a Windows issue this is set to 0 there for now
       "total" string => ifelse("windows", "0",
                                $(mon.value_mem_total)),
       meta => { "inventory", "attribute_name=Memory size (MB)" };
 
+    enterprise::
       "free" string => ifelse("windows", "0",
                               $(mon.value_mem_free)),
       meta => { "report" };
+
+    enterprise.aix::
+      "total" string => execresult("/usr/bin/lparstat -i | grep 'Online Memory' | awk '{ print $4 }'", "useshell"),
```

Comment:
If we use this execresult to collect memory let's get rid of grep.

This should work:

`/usr/bin/lparstat -i | awk '/Online Memory/ {print $4 }'

Reply:
@ncharles can you update the PR to drop the extra pipeline?

```
$ lparstat -i | grep Online
Online Virtual CPUs                        : 3
Online Memory                              : 2048 MB
```

```
$ /usr/bin/lparstat -i | grep 'Online Memory' | awk '{ print $4 }'
2048
```

Also, it would be nice if we had the information about what cf-monitord was capturing for memory.

```
$(mon.value_mem_free)
```

Reply:
oh TOTAL not free, i was looking at the wrong line.